main.go

```diff
@@ -0,0 +1,127 @@
+package authenticated_block_store
+
+import (
+	"fmt"
+	"log"
+	"io/ioutil"
+        "strings"
+        "os"
+        "crypto/sha256"
+        "math/rand"
+        "time"
```

Comment:
Why are these indented weirdly? Try running gofmt over all your code, that should fix this (and probably make some other changes which we also want)